commit 117a58635afba7a00463285a8d657a14838b1b10 (refs/changes/36/2336/3)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2017-08-08T17:45:19+00:00 (2 years, 2 months ago)
    
    VOLAPI-65 rename required_nfs_volumes VM objects' property to volumes
    Reviewed-by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/volapi-updater.js b/volapi-updater.js
index 1eeead6..95ddf84 100644
--- a/volapi-updater.js
+++ b/volapi-updater.js
@@ -922,7 +922,7 @@ function updateReferencesAndReservationsForVm(vmUuid, options, callback) {
                 }
 
                 if (ctx.vm === undefined ||
-                    ctx.vm.required_nfs_volumes === undefined) {
+                    ctx.vm.volumes === undefined) {
                     next();
                     return;
                 }
@@ -930,7 +930,7 @@ function updateReferencesAndReservationsForVm(vmUuid, options, callback) {
                 volumeOwnerUuid = ctx.vm.owner_uuid;
 
                 log.info({
-                    volumes: ctx.vm.required_nfs_volumes,
+                    volumes: ctx.vm.volumes,
                     vm: ctx.vm
                 }, 'Loading volumes required by VM');
 
@@ -981,7 +981,7 @@ function updateReferencesAndReservationsForVm(vmUuid, options, callback) {
                             done();
                         });
                     },
-                    inputs: ctx.vm.required_nfs_volumes
+                    inputs: ctx.vm.volumes
                 }, function onRequiredVolsLoaded(loadErr) {
                     ctx.volumesToProcess = requiredVolumes;
                     next(loadErr);
